src/FLEXDUST.f90

```diff
@@ -347,16 +350,48 @@ program FLEXDUST
                     !In first time step only determine erodibility at grid point if switched on
                     !********************************************************
                     if(tot_sec.eq.0 .and. EROSION_TOPO)then
+                        
+                        erodibility_total(ix,iy)=0
```

Comment:
Something similar is now implemented in the main repository, but using a loop based on regions defined in dust_mod.f90;

                            do dd = 1, size(topo_scale_erosion,1)
                                !get lower left corner (ix_ll, iy_ll) of erosion area
                                call getGridPointWind(lat_out-(topo_scale_erosion(dd)/2.), lon_out-(topo_scale_erosion(dd)/2.), &
                                                        dummy_int, ix_ll, iy_ll, dummy_int, dummy_int)
                                !get upper right corner (ix_ur, iy_ur) of erosion area
                                call getGridPointWind(lat_out+(topo_scale_erosion(dd)/2.), lon_out+(topo_scale_erosion(dd)/2.), &
                                                        dummy_int, ix_ur, iy_ur,dummy_int,dummy_int)
                                !scale erodibility in this area
                                call getErodibility(tmp_erosion, ix_wind(ix), iy_wind(iy), ix_ll, ix_ur, iy_ll, iy_ur)

                                erodibility(ix,iy)=erodibility(ix,iy)+real(tmp_erosion/real(size(topo_scale_erosion,1)))
                            end do